timescaledb/overview/how-does-it-compare/timescaledb-vs-postgres.md

```diff
@@ -1,251 +1,197 @@
 
-TimescaleDB achieves a much higher and more stable ingest rate than
-PostgreSQL for time-series data. As described in our [architectural discussion][],
-PostgreSQL's performance begins to significantly suffer as soon as indexed tables
-can no longer fit in memory.
+## Better performance at scale
+
+TimescaleDB performs orders of magnitude better at high data volumes. Your
+applications are future-proof even as they grow rapidly.
 
-In particular, whenever a new row is inserted, the database needs to
-update the indexes (for example, B-trees) for each of the table's indexed
-columns, which involves swapping one or more pages in from disk.
-Throwing more memory at the problem only delays the inevitable, and
-your throughput in the 10K-100K+ rows per second can crash to
-hundreds of rows per second once your time-series table is in the tens
-of millions of rows.
+### 1000 times faster performance for time-series queries
 
-TimescaleDB solves this through its heavy utilization of
-time-space partitioning, even when running _on a single machine_.  So
-all writes to recent time intervals are only to tables that remain in
-memory, and updating any secondary indexes is also fast as a result.
+We tested the performance of TimescaleDB + PostgreSQL against PostgreSQL alone,
+using different time-based queries. We used one month's worth of data, which
+amounted to one billion rows organized into four-hour partitions. Running 100
+queries at a time, TimescaleDB + PostgreSQL consistently outperformed a standard
+PostgreSQL database. Queries are up to 1000&nbsp;times faster.
 
-Benchmarking shows the clear advantage of this approach. The
-following benchmark out to 1 billion rows (on a single machine)
-emulates a common monitoring
-scenario, with database clients inserting moderately sized batches of
-data containing time, a device's tagset, and multiple numeric metrics (in
-this case, 10).  Here, experiments were performed on a standard Azure VM
-(DS4 v2, 8 core) with network-attached SSD storage.
+This table shows the query latency of PostgreSQL compared to TimescaleDB +
+PostgreSQL. The data uses PostgreSQL 14 and TimescaleDB 2.7. For more
+information on the comparison, see the [comparison blog
+post][blog-postgresql-vs-timescaledb].
+
+<img class="main-content__illustration"
+src="https://www.timescale.com/blog/content/images/2022/09/Query-latency-deep-dive--1--1.png"
+alt="PostgreSQL query latency in milliseconds, compared to TimescaleDB. For 14
+query statements, TimescaleDB performs faster in 13 cases, with improvement
+ranging from 4 times to 1031 times. In the thirteenth case, TimescaleDB performs
+slightly worse, at 0.8 times as fast as standard PostgreSQL." />
 
-<!-- vale Google.Units = NO -->
+TimescaleDB achieves this performance by using [hypertables][hypertables]. With
+hypertables, your data is seamlessly and automatically partitioned by time, but
+you get the ergonomic experience of interacting with a single, virtual table.
```

Comment:
```suggestion
you get the ergonomic experience of interacting with a single, virtual table.
```

Is "ergonomic" the right word here?

Reply:
"Ergnomic developer experience" is something I've heard, but happy to change if it sounds a bit niche or weird. I'll think up a good alternative!

Reply:
I dunno, I hear "ergonomic" and think of office chairs 🤷‍♀️ Maybe that's just me though?